Tests/SonarScanner.MSBuild.Tasks.UnitTest/GetAnalyzerSettingsTests.cs

```diff
@@ -181,7 +181,7 @@ public void ConfigExists_Legacy_SettingsOverwritten()
             testSubject.RuleSetFilePath.Should().Be("f:\\yyy.ruleset");
             testSubject.AnalyzerFilePaths.Should().BeEquivalentTo("c:\\analyzer1.DLL", "d:\\analyzer2.dll", "e:\\analyzer3.dll");
             testSubject.AdditionalFilePaths.Should().BeEquivalentTo("c:\\add1.txt", "d:\\add2.txt",
-                "e:\\subdir\\add3.txt", "original.should.be.preserved.txt");
+                "e:\\subdir\\add3.txt", "original.should.be.preserved.txt", "original.should.not.be.replaced\\add2.txt", "e://foo//should.not.be.replaced//add3.txt");
```

Comment:
what is the semantic difference between `original.should.be.preserved` and `original.should.not.be.replaced` ?

Reply:
My original idea was to "save the evolution" and see what was once replaced and not to replace it anymore, however now if I think again it will just create confusion so I am renaming them all to  `original.should.be.preserved`.